src/main/java/com/optimaize/langdetect/LanguageDetectorImpl.java

```diff
@@ -199,11 +196,11 @@
     private double[] initProbability() {
         double[] prob = new double[ngramFrequencyData.getLanguageList().size()];
         if (priorMap != null) {
-            //TODO analyze and optimize this code, looks like double copy.
             System.arraycopy(priorMap, 0, prob, 0, prob.length);
-            for(int i=0;i<prob.length;++i) prob[i] = priorMap[i];
```

Comment:
If the removal of this line is intentional, you should update the Javadoc for this method too.